### PR TITLE
chore: add issue template

### PR DESCRIPTION
**Pull request by yuluo-yx**
- For: #373 issue tmpl

### CODE REVIEW

**Review by LinuxSuRen — APPROVED** [review]
LGTM and thanks for your work.

File: .github/ISSUE_TEMPLATE/non-crash-bog.md
Comment (LinuxSuRen) [inline .github/ISSUE_TEMPLATE/non-crash-bog.md:1]: I'm wondering if this is a typo `bog`. Should it be `bug`?

Comment (yuluo-yx) [reply .github/ISSUE_TEMPLATE/non-crash-bog.md:1]: done!

**Comment by yuluo-yx** [comment]
ptal @Agility6

**Review by Agility6 — APPROVED** [review]

**Review by Agility6 — APPROVED** [review]

**Comment by LinuxSuRen** [comment]
Thanks two of you.

**Pull request merged**